### ISSUE TITLE
Action Button

### ISSUE THREAD

**Issue by andrefarrugia96** [RyanDelia/IMTAssignment2016#10]
Will you be adding another action button to the quiz game?

**Comment by RyanDelia** [comment]
Yes of course i am thinking of adding and exit button during the game so if the user wants to stop playing the game he does not have to Finnish the game to exit.
